package/kernel/mac80211/patches/subsys/341-mac80211-fix-legacy-rate-decode.patch

```diff
@@ -0,0 +1,14 @@
+Index: backports-6.1.24/net/mac80211/sta_info.c
```

Comment:
Please refresh this patch using
```
make package/mac80211/{clean,refresh}
```

Reply:
Thanks for your review and instructions on how to refresh the patch.
I squashed the changes and pushed the new version.